### PR TITLE
0.13 example update: jib-container

### LINKED ISSUE #3782: 0.13 example update: jib-container

### PR DESCRIPTION
**Pull request by vvagaytsev**
**What this PR does / why we need it**:
Some code fixes and example updates.

**Which issue(s) this PR fixes**:

Fixes #3782

**Special notes for your reviewer**:

### CODE REVIEW

File: examples/jib-container/helloworld/helloworld.garden.yml [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
   dockerBuild: ${environment.name == "local"}
-services:
-  - name: helloworld
+
+---
+kind: Deploy
+type: container
+name: helloworld
+build: helloworld
+spec: { }
```
Comment (Orzelius) [inline examples/jib-container/helloworld/helloworld.garden.yml:15]: Is this needed?

**Review by Orzelius — APPROVED** [review]

**Review by Walther — COMMENTED** [review]
👍 
We should also re-enable `e2e-jib-container` in the circleci config

File: examples/jib-container/helloworld/helloworld.garden.yml (hunk 1 above)
Comment (vvagaytsev) [reply examples/jib-container/helloworld/helloworld.garden.yml]: No, it's not necessary, I removed it. Thanks! :)

**Review by Walther — APPROVED** [review]
🚢

**Pull request merged**